src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,91 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int RESIZE_MULTIPLIER = 2;
+    private int capacity;
```

Comment:
We don't need capacity as a field property

Reply:
It can be local variable in resize()

Reply:
I dont understand, which capacity do you mean

Reply:
on line 7 it can be local variable in method

Reply:
I dont understand you, we need field capacity in each method